pipeline/inspection.py

```diff
@@ -54,6 +55,29 @@ def convert_value(cls, v):
             return extract_first_number(v)
         return None
 
+# class syntax
+
+class RegistrationNumberType(str, Enum):
```

Comment:
Do we need another class for registration number type? How about having a enum field in the RegistarionNumber class instead?

Reply:
Not necessary but it's more hints for the model.

Reply:
I think it's not necessary to have a separate class just for RegistrationNumberType. At least now that it doesn't have multiple fields.